packages/salesforcedx-vscode-core/src/scratch-org-decorator.ts

```diff
@@ -6,10 +6,13 @@
  */
 
 import * as fs from 'fs';
+import * as os from 'os';
 import * as path from 'path';
 import { StatusBarAlignment, StatusBarItem, window, workspace } from 'vscode';
 
-const CONFIG_FILE = path.join(workspace.rootPath!, '.sfdx/sfdx-config.json');
+const CONFIG_FILE = workspace.rootPath
```

Comment:
Since the command can be activated without a workspace, the non-null assertion would cause an error. I set it to the top level sfdx-config.json. If this file does not exist, we will still be okay since displayDefaultUsername essentially ignores files that do not exist.